ale_linters/go/gometalinter.vim

```diff
@@ -12,6 +13,13 @@ function! ale_linters#go#gometalinter#GetCommand(buffer) abort
     let l:executable = ale_linters#go#gometalinter#GetExecutable(a:buffer)
     let l:filename = expand('#' . a:buffer)
     let l:options = ale#Var(a:buffer, 'go_gometalinter_options')
+    let l:lint_package = ale#Var(a:buffer, 'go_gometalinter_lint_package')
```

Comment:
Same here... Maybe just call it `l:package`?

Reply:
without the verb `lint` it becomes unclear what the variable is used for.